pkg/controllers/status/cluster_status_controller.go

```diff
@@ -278,6 +278,7 @@ func (c *ClusterStatusController) updateStatusIfNeeded(cluster *clusterv1alpha1.
 		klog.V(4).Infof("Start to update cluster status: %s", cluster.Name)
 		err := retry.RetryOnConflict(retry.DefaultRetry, func() (err error) {
 			_, err = helper.UpdateStatus(context.Background(), c.Client, cluster, func() error {
+				currentClusterStatus.RemedyActions = cluster.Status.RemedyActions
```

Comment:
I'm afraid this would make code hard to maintain because you'd need to know exactly which field isn't managed by the cluster-status-controller.

Reply:
Is it better to list the fields that need to be updated in a forward?